debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/history/JdbcSchemaHistory.java

```diff
@@ -167,16 +169,29 @@ protected synchronized void recoverRecords(Consumer<HistoryRecord> records) {
                         try (
                                 Statement stmt = conn.createStatement();
                                 ResultSet rs = stmt.executeQuery(config.getTableSelect())) {
+                            StringBuilder sb = new StringBuilder();
+                            int currentRecordSeq = Integer.MAX_VALUE;
                             while (rs.next()) {
+                                int recordSeq = rs.getInt("record_insert_seq");
```

Comment:
The column names should be configurable (ithis is oversight in "history_data") too.